altair/utils/_importers.py

```diff
@@ -0,0 +1,82 @@
+from types import ModuleType
+from packaging.version import Version
+
+
+def import_vegafusion() -> ModuleType:
+    min_vf_version = "1.4.0"
+    try:
+        import vegafusion as vf  # type: ignore
+
+        if Version(vf.__version__) < Version(min_vf_version):
+            raise ImportError(
+                f"The vegafusion package must be version {min_vf_version} or greater. "
+                f"Found version {vf.__version__}"
+            )
+        return vf
+    except ImportError as err:
+        raise ImportError(
+            'The "vegafusion" data transformer and chart.transformed_data feature requires\n'
+            f"version {min_vf_version} or greater of the 'vegafusion-python-embed' and 'vegafusion' packages.\n"
+            "These can be installed with pip using:\n"
+            f'    pip install "vegafusion[embed]>={min_vf_version}"\n'
+            "Or with conda using:\n"
+            f'    conda install -c conda-forge "vegafusion-python-embed>={min_vf_version}" '
+            f'"vegafusion>={min_vf_version}"\n\n'
+            f"ImportError: {err.args[0]}"
+        ) from err
+
+
+def import_vl_convert() -> ModuleType:
+    min_vlc_version = "0.12.0"
```

Comment:
Is this version number determined by what Vega-Lite version is bundled in VLC? If yes, could you add a bullet point to the `Updating the Vega-Lite version` section in `NOTES_FOR_MAINTAINERS.md` to update this version number every time we update Vega-Lite?